database.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE family;
+
+USE family;
+
+CREATE TABLE `users` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`name` VARCHAR(25) NOT NULL,
+	`status` VARCHAR(50) NOT NULL,
+	`age` INT(4) NOT NULL,
+	`address` VARCHAR(100) NOT NULL,
+	`approve_status` TINYINT (1) NOT NULL,
+	`password` VARCHAR(100) NOT NULL,
+	`image` VARCHAR(100),
+	PRIMARY KEY (`id`)
+);
+
+CREATE TABLE `tasks` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`time_created` INT(10) NOT NULL,
+	`created_by` VARCHAR(25) NOT NULL,
+	`executor`  VARCHAR(25) NOT NULL,
+	`task` TEXT NOT NULL,
+	`status` VARCHAR(15) NOT NULL,
+	`time_start` INT(10) NOT NULL,
+	`time_end` INT(10) NOT NULL,
+	`comment` TINYTEXT NOT NULL,
```

Comment:
tinytext? what if i wanna write a lot of words and sentences? 
maybe i'm gonna write a novel here?
I guess it's a pretty good idea, hm...